camera_topic_name: /equirectangular/image_raw
# model_path: $(find camera_apps)/model
conf_threshold: 0.4
# conf_threshold: 0.3
